libs/community/langchain_community/embeddings/huggingface.py

```diff
@@ -7,7 +7,7 @@
 
 DEFAULT_MODEL_NAME = "sentence-transformers/all-mpnet-base-v2"
 DEFAULT_INSTRUCT_MODEL = "hkunlp/instructor-large"
-DEFAULT_BGE_MODEL = "BAAI/bge-large-en"
+DEFAULT_BGE_MODEL = "BAAI/bge-large-en-v1.5"
```

Comment:
updating the default model is technically a breaking change, could we stick to just updating all the docs?

Reply:
I'm fine with that for now but is there a ballpark plan for eventually doing an update?

Reply:
in general we've moved away from having default values for the model arg precisely because of this (over time it either becomes outdated or you have to make a breaking change). 

in this case think best first step would be to always explicitly pass in model name in all the documentation, and in a future minor version bump we can remove the default altogether

Reply:
That makes sense to me; let me change this to a docs update and a deprecation warning

Reply:
@baskaryan this should be all set for another review; I kept the scope narrow to mostly just huggingface embeddings (and related docs) for this first go at it.